descriptors/ModuleDescriptor-template.json

```diff
@@ -62,7 +62,7 @@
     },
     {
       "id": "loan-storage",
-      "version": "7.3",
+      "version": "7.4",
```

Comment:
Are there any modules that have a dependency on loan-storage v7.3 or loan-policy-storage v2.3.

Reply:
Right, there can be dependencies that we don't know of. FWIW we _do_ know that mod-circulation has a dependency on loan-storage 7.3.